server/lib/chart.js

```diff
@@ -0,0 +1,127 @@
+const _ = require('lodash');
+const util = require('./util');
+
+module.exports = {
+  barChar,
+};
+
+/**
+ * @function barChar
+ * This function help to skip some manipulation for respecting chartjs lib format...
+ */
+
+function barChar(params) {
+  const {
+    label, data, item, yAxesLabelString,
+    xAxesLabelString, canvasId,
+  } = params;
+
+  const chart = {
+    labels : [],
+    data : [],
+  };
+  const serieMap = {};
+
+  const dailyData = _.groupBy(data, label);
```

Comment:
It seems to me that you probably want to either:
  1. Sum based if you are dealing with quantities (using [_.sumBy()](https://lodash.com/docs/4.17.15#sumBy))
  2. Count the number of records if you are looking at just records (using [_.countBy()](https://lodash.com/docs/4.17.15#countBy)).

But maybe I'm missing something.